src/foam/u2/dialog/ApplicationPopup.js

```diff
@@ -234,10 +247,21 @@ foam.CLASS({
               }))
             .end()
           .end()
-          .add(this.slot(function (content$childNodes) {
+          .add(this.slot(function(progressView) {
+            return this.E()
+              .tag(progressView, {
+                progressMax$: self.progressMax$,
+                progressValue$: self.progressValue$
+              });
+          }))
+          .add(this.slot(function(content$childNodes) {
             if ( ! content$childNodes ) return;
             let title = '';
             for ( const child of content$childNodes ) {
+              if ( foam.u2.Progressable.isInstance(child.data) ) {
```

Comment:
Since we don't make use of the `controlBorder` to lift this indicator, the wizard itself cannot be wrapped in any composite view or additional border.